src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/mappers/DateMapper.java

```diff
@@ -33,4 +33,15 @@ public static int secondsSinceStartOfTime(ZonedDateTime startOfTime, Instant ins
     public static LocalDateTime asDateTime(LocalDate localDate, int secondsSinceStartOfDay) {
         return localDate.atStartOfDay().plusSeconds(secondsSinceStartOfDay);
     }
+
```

Comment:
Could you add a comment on the above method on why we skip the noon - 12 hours calculation here. Comment:
```
// In OTP LocalDate is sometimes used to represent ServiceDate. This calculation is "safe" because 
// calculations on LocalDate ignore TimeZone adjustments, just like the ServiceDate. So, in this case it is not 
// necessary to: 'NOON - 12 hours + secondsSinceStartOfDay'
```

This method would also be clearer if the variable names was `serviceDate` and `timeInSeconds`. To me the `secondsSinceStartOfDay` means midnight, which is not trough given a timeZone and DST switch.